packages/ui/Stepper.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+
+type DefaultStep = {
+  title: string;
+};
+
+function Stepper<T extends DefaultStep>(props: { href: string; step: number; steps: T[] }) {
```

Comment:
Put it in `/v2/` for now? Also let's try to hurry with #3051

Reply:
We can't move, see comment above.